### DISCUSSION TITLE
WEMI to WEMI Relationships Questions: Added entries

### DISCUSSION THREAD

**Discussion by tmqdeborah** [crystalyragui/MARC2RDA#455] [General]
### Here are questions about mapping WEMI to WEMI 'associated' relationship (not primary relationship) values that are found in added entry fields.

These fields include:
- subject added entries with title data: 600-630 
- non-analytical added entries with title data: 700-730 & 740 
- series added entries: 800-830 & 440 (obsolete)

******* Please reply to each question separately, using "**Write a reply**" instead of starting a new comment. *******

Jump to:
NEW:
- Q: [How can we tell whether a series added entry applies to an augmented single work vs its aggregating work?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10329179)

-----
- Q: [What should we do about WEMI that are related to an associated WEMI?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-9763948) **A: Add a disclaimer to documentation**--To be done
- Q: [Should we map Unconstrained RDA labels and URIs in the WEMI Relationships Table](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-9783501)? **A: Yes--Done**
- Q: [Should we provide a map of RDA labels that have changed over time?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-9783645)  **A: Yes**--To be done
- Q: [Should we map PCC and/or LC labels and URIs to RDA URIs in the WEMI Relationships Table?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-9822204) **A: No**
- Q: [What should we do with 6XX headings fields that contain $e or $4?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-9895470) **A: Do not map the subfields**--To be added to "Using" instructions
- Q: [What should we do with 7XX name/title added entry fields that contain $e, or that have $4 that contain Agent relator values?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10002218) **A: To be decided**
- Q: [Is it OK if an access point for a work has the same Nomen string as an access point for its expression?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10002990) **A: Undetermined; see next question**
- Q: [What should we use as the default relationship between a WEMI and an associated WEMI when we have no recorded or mapped RDA URI to use?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10067307) **A: related work of manifestation**--To be added to "Using" instructions
- Q: [Question: How can we tell whether a series added entry applies to an augmented single work vs its aggregating work?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10329179) **A: Apply a series added entry to the aggregating work. (DF)**

- Q: What should we do when a record contains multiple 7XX added entry fields for the same work or expression?

**Comment by tmqdeborah** [comment]
### Question: Is it OK to make a relationship between a resource WEMI and an associated WEMI (that is provided in a heading of an added entry field) when we do not have enough information to describe and relate any WEMIs that are related to those associated WEMIs?
For further details, see [Phase I Project Plan Refined Diagrams DRAFT s. 13-16](https://docs.google.com/presentation/d/1sdBMcy4BTALBWUAxWY2Gyg_4RlvJpuja/edit?pli=1#slide=id.p13)

The consensus at the meeting on July 6, 2024 was that we should:
- Mint an IRI for an associated WEMI provided in an added entry heading and relate it to the appropriate resource WEMI
- Mint an IRI for an Agent (if one is provided in the heading) and relate it to the associated WEMI relationship
- Add a disclaimer in the transform documentation (somewhere) to explain that this is not fully conformant with RDA resource description guidance.

@GordonDunsire could you please repeat the wording that you suggested for this disclaimer?

**Reply by GordonDunsire** [reply]
Assuming that the three contexts given are exhaustive, or alternatively saying that the entity identified in the heading of a related entry field is not part of the resource that is described in the M21 record (i.e. does not have a primary relationship to the manifestation, which is what the diagram shows), then the disclaimer is: A non-primary related work, expression, manifestation, or item that is minted from a related entry heading may not conform to the RDA requirement that its own immediate primary entities are identified and related. It will conform to the requirement that it is linked to at least one of its appellations.

Note that I don't think that "part" related work and expressions entries are relevant; either the whole or a part has a primary relationship within the resource described, but not both. I also think that, in general, a related expression heading provides enough information to mint its work with an appellation, as well as a creator agent with an appellation.

In short, the essential consideration focuses on the manifestation described in the record: does it embody the related entity (i.e. work or expression), or is it exemplifed by the related item, or is it a whole or part of the manifestation? Is this correct?

**Reply by tmqdeborah** [reply]
@GordonDunsire, thanks for the wording for the disclaimer; it will be very helpful when the time comes to add it to the technical documentation for the project.

Thanks also for reminding me that “in general, a related expression heading provides enough information to mint its work with an appellation, as well as a creator agent with an appellation.” I have added this to slides 13-16. 

If I am understanding you correctly, you have raised the issue of whether a relationship for a ‘part WEMI’ is a resource/primary relationship or an associated relationship. We haven’t really touched on whole/part expression/work relationships yet, but we can at least make a start now. I am going to start a separate comment for this, with what I'm thinking at this point.

We also have not looked at how to handle multi-part manifestations yet (i.e., "is it a whole or part of the manifestation"?) I will start a separate comment for this, also, at some point. This will bring up the Linking entry fields:
773 - Host Item Entry (R)  [Full](https://www.loc.gov/marc/bibliographic/bd773.html) 
774 - Constituent Unit Entry (R)  [Full](https://www.loc.gov/marc/bibliographic/bd774.html)

**Comment by tmqdeborah** [comment]
### Q: Should we map Unconstrained RDA labels and URIs in the [WEMI Relationships Table](https://docs.google.com/spreadsheets/d/1eZysF8vc-fjWHn90OaTw1ZAr7_OiZdId/edit#gid=334496971)?

  - We did this for [Agent Relationships](https://docs.google.com/spreadsheets/d/1Z_AqsuVULXvihLuCS9Xrlw6OGIeWCXmQ/edit?rtpof=true).
-  I have added them to the WEMI table because it was easy for Richard to write a script to do so, using the mapping provided at RDA Registry [mapRDA2Unc.html](http://www.rdaregistry.info/Maps/mapRDA2Unc.html) (so there is no need for us to check that Uncon definitions match RDA definitions.
  - As we did for Agent Relationships, we need to code the transform to use a default relationship if we are not able to determine the domain or the range because the unconstrained properties map to multiple domains. 
  - Consensus in the July 6, 2024 meeting seemed to be that we should include them

**Reply by GordonDunsire** [reply]
I think I suggested that Expression should be the default related WEMI entity; the heading is treated as an access point, and it can be qualified in the transform by a content type found elsewhere in the record. The main issue is the domain, before the relationship and related entity :-)

**Reply by tmqdeborah** [reply]
For an unconstrained WEMI-WEMI relationship label or URI that maps to multiple domains, we can’t determine the domain (just as we could not do for the WEMI-Agent relationships). But there are a few unconstrained relationships that do map to only one domain and so can be mapped. So, I think it is worth mapping those unconstrained RDA labels and URIs.

Yes, sorry, I should have added that we did agree that we would use Expression as the default range and Manifestation as the default domain) for headings from 700-730 added entry fields. Although that options has its problems, e.g., the Expression qualifier issue that you have raised.

We can use the “content type found elsewhere in the record” as a qualifier for a heading for an expression with a **primary/resource relationship**. So, we will be able to add Expression qualifiers that are not provided in an ‘analytical added entry’ (700Ind2=2) heading for a primary relationship. But this table is for relating headings for ‘**associated**’ WEMI entities to appropriate resource WEMI entities, and the content type of the resource expression might very well not be the same as the content type or language, etc. of the associated expression.  

So, we will not be able to add Expression qualifiers that are not already provided in a heading for ‘non-analytical added entries’ (7XXInd2 not=2).

**Comment by tmqdeborah** [comment]
### Q: Should we provide a map of RDA labels that have changed over time? Is there a list of those changes?

A large number of labels were changed in the RDA Registry in 2015 but the ‘Original RDA’ labels have been used in legacy records and are still being used. A few examples are:
- ‘reviewed in manifestation’ changed to ‘manifestation reviewed by’ 
- ‘adapted as video game (work)’ changed to ‘adapted as video game work’ – but parentheses can be ignored in mapping
- and more recently, ‘incumbent’ changed to ‘officer’

And then there are the relationships that have been deprecated, rather than revised, e.g., ‘Container of (work)’ which LC says is now ‘[part o](https://id.loc.gov/vocabulary/relationship/partof.html)f’ (A work or an expression that has another work or expression as a discrete component. A part work is considered to be essential to the whole work). However, ‘container of (work)’ has been used extensively for aggregated works; see [7XX$eContainer of_UW.txt](https://drive.google.com/drive/folders/1PVipw3x09XQy7-ZLSfQCiOv2nL9TR1u1)

Catalogers used these labels in good faith; so, I think it would be good if we could find a way to map them, if we can find the ones that were changed or deprecated and replaced (e.g., because of a change in domain or range) and are able to find the replacements for them.

**Reply by GordonDunsire** [reply]
There's a bunch of stuff in the Registry: [http://www.rdaregistry.info/rgArch/](http://www.rdaregistry.info/rgArch/).

But is this a priority? Phase 1 time is running out ...

**Comment by tmqdeborah** [comment]
### Question: Should we map PCC and/or LC labels and URIs to RDA URIs in the [WEMI Relationships Table](https://docs.google.com/spreadsheets/d/1eZysF8vc-fjWHn90OaTw1ZAr7_OiZdId/edit#gid=334496971)?

I have put two files in the Headings Fields Examples folder. They show the number of records and examples of those records that contain labels that are not either RDA or Unconstrained:
- [700$iNotRdaOrUnconRelationships_UW.txt](https://drive.google.com/drive/folders/1m5tP46eD2E00T5RBQ5qfOwzeDXvRANAn)
- [700$iNotRdaOrUnconRelationships_LC.txt](https://drive.google.com/drive/folders/1m5tP46eD2E00T5RBQ5qfOwzeDXvRANAn)

I have started a table of the LC Relationship labels (complete) and PCC Relationship labels (incomplete) that apply to WEMI  (for 700 fields only, so far). That table is at: [LC PCC Relationship labels.20240615.xlsx](https://drive.google.com/drive/folders/1PVipw3x09XQy7-ZLSfQCiOv2nL9TR1u1)

- [PCC Relationship labels](https://www.loc.gov/aba/rda/mgd/relationshipLabels/index.html)
  - Are PCC labels always the same as the RDA Unconstrained labels that are provided in the WEMI table?
   - Are the definitions for PCC labels always exactly the same as for the RDA labels?
     - If we decide that these PCC terms should be mapped where possible, then someone will have to add the remaining terms (where the definitions match)

  - N.B.
    - PCC says that relationships with Manifestation as the range are “not currently possible because authorized access points for manifestations are not made in PCC practice” --https://www.loc.gov/aba/rda/mgd/relationshipLabels/mg-rl-work-manifestation.pdf
    - Relationships with Item as the range are “not currently possible because authorized access points for item are not made in PCC practice” --https://www.loc.gov/aba/rda/mgd/relationshipLabels/mg-rl-work-item.pdf


- [LC Relationship terms](https://id.loc.gov/vocabulary/relationship.html)
  - LC provides Work and Expression labels only 
  - These terms are not always the same as either the [PCC Relationship labels](https://www.loc.gov/aba/rda/mgd/relationshipLabels/index.html) or the RDA Unconstrained labels (e.g., ‘abstracted as’ vs. ‘abstract’)
  - We would need to check that all of the definitions for the LC labels are exactly the same as the RDA terms

**Comment by tmqdeborah** [comment]
### Question: What should we do with 6XX headings fields that contain $e or $4?

Subfields $e and $4 are valid in 6XX fields. However, examples of the use of those fields are puzzling. For example: 
600 10 $a Sidney, Algernon, $d 1623-1683, $e defendant.
600 10 $a Weinzweig, John, $d 1913-2006, $e interviewee.
610 20 $a University of Washington. $b Ethnomusicology Archives, $e curator.
600 10 $a Edman, Irwin, $d 1896- $e ed.
600 10 $a Kirk, Kenneth E. $q (Kenneth Escott), $d 1886-1954, $e ed. $t Apostolic ministry.

Are these examples supposed to mean that the subject of the work is the agent as defendant, or interviewee, or curator, or editor? If not and they mean that the agent is the defendant of, or interviewee of, or collector (curator) of a work, or editor of an expression then, these subfields belong in an appropriate 7XX field instead of a 6XX field.

This question was discussed in our June 27, 2024 meeting and it was determined that only two subfield $e or $4 relationship values are truly subject relationships:
- $e depicted; $4 dpc
- $e setting; $4 stg

Both of these LC labels/codes are mapped to RDA label ‘subject’, and the default relationship for all 6XX fields is ‘subject’.

**Reply by tmqdeborah** [reply]
DECIDED: Do not map $e or $4 values found in 6XX fields.

**Comment by tmqdeborah** [comment]
### Question: What should we do with 7XX name/title added entry fields that contain $e, or that have $4 that contain Agent relator values?

Subfields $e and $4 are technically valid in 7XX fields. And it seems like it might possibly be useful to use them in 7XX name/title fields to indicate the relationship between the Agent (name) and the WE (title) that are described by a headings field. For example:
700 1  $a Reeser, Eduard, $d 1908-2002, $e ed. $t Music in Holland.
_Means that Reeser is the editor of the expression “Music in Holland” (OR the aggregator of the aggregating work?)_

However, catalogers have been instructed that they should not add agent relator values in 7XX name/title fields. @AdamSchiff, could you summarize why that guidance is given, and provide a link to it?

Despite this guidance, records have been found with:
- Agent relator codes in 7XX name/title fields: <[700$t AND $4_AgentLCRelatorCodes_LC](https://drive.google.com/drive/folders/1m5tP46eD2E00T5RBQ5qfOwzeDXvRANAn)>
- Agent relator terms in 7XX name/title fields: <[700$t AND $e_AgentLCRelatorTerms_LC](https://drive.google.com/drive/folders/1m5tP46eD2E00T5RBQ5qfOwzeDXvRANAn)>

Should we:
- ignore $e and $4 values in 7XX name/title added entry fields (as we do for 6XX fields)?
- use $e values and the [Agent Relators Table](https://docs.google.com/spreadsheets/d/1Z_AqsuVULXvihLuCS9Xrlw6OGIeWCXmQ/edit?gid=1079470556#gid=1079470556) to map a relationship between the Agent and the WE described by a headings field?
- use $4 values and the [Agent Relators Table](https://docs.google.com/spreadsheets/d/1Z_AqsuVULXvihLuCS9Xrlw6OGIeWCXmQ/edit?gid=1079470556#gid=1079470556) to map a relationship between the Agent and the WE described by a headings field? 
  - We would have to check a $4 twice: 
    -  Against the [Agent Relators Table](https://docs.google.com/spreadsheets/d/1Z_AqsuVULXvihLuCS9Xrlw6OGIeWCXmQ/edit?gid=1079470556#gid=1079470556) for the relationship between the Agent and the WE described by a headings field
    - Against the [WEMI Relationships Table](https://docs.google.com/spreadsheets/d/1eZysF8vc-fjWHn90OaTw1ZAr7_OiZdId/edit?gid=334496971#gid=334496971) for the relationship between the WE in the headings field and the WEMI described by the record

**Reply by GordonDunsire** [reply]
@tmqdeborah: In the Reeser example, what leads you to suggest that subfield $t 'Music in Holland' is an expression? If this is the default preference because there is no information in the 'work/title' part of the heading, then the default should be conditional on the relationship. Subfield $e 'ed.' is notorious, not least as indicated by your alternative interpretations. I suggest that if 'ed.' or its cognate forms, is recorded as the relationship between 'name' and 'title' from a heading then the 'title' part should be treated as a work.

Subfield $4 'specifies the relationship from the entity described in the record to the entity referenced in the field'. It is similar to subfield $i 'Designation of a relationship between the resource described in the 7XX field and the resource described in the 1XX/245 of the record'. Neither of these is a relationship between the 'name' and 'title' parts of a heading.

Surely @AdamSchiff is referring to subfield $e, but not to subfield $4?

**Reply by tmqdeborah** [reply]
@GordonDunsire, 
Good point, we could write special coding to always treat a $e that contains 'ed.' or 'comp.' as an 'aggregator' relationship (rather than referring to the Agent Relators Table) and therefore consider the heading to be an aggregating work. This is if we decide that we will **not** ignore $e and $4 subfields (that contain WEMI-Agent relator values) in name/title headings.

Subfield $4 is indeed defined as "Code or URI that specifies the relationship from the entity described in the record to the entity referenced in the field." However it can be used for either a WEMI-Agent relationship ("A source of relationship codes is: [MARC Code List for Relators](https://www.loc.gov/marc/relators/)") or a WEMI-WEMI relationship (mirroring a $i). There is probably NACO guidance that says we should not use $4 for a WEMI-Agent relationship in a name/title heading, but it has been used for that purpose, see [700$t AND $4_AgentLCRelatorCodes_LC](https://drive.google.com/drive/folders/1m5tP46eD2E00T5RBQ5qfOwzeDXvRANAn)

**Reply by cspayne** [reply]
Tentative decision based on 2024-07-10 M2R meeting: We are considering keeping $e and $4 relationships between these minted agents and works and expressions

**Comment by tmqdeborah** [comment]
### Question: Is it OK if an access point for a work has the same Nomen string as an access point for its expression? 

We are often not going to be able to tell whether a 7XX or a 6XX name/title heading is for:
-	an aggregating expression/work (e.g., $a Lewis, C.S. $t Chronicles of Narnia. $l French)
-	a multi-part expression/work (e.g., $a Tolkien, J.R.R. $t Lord of the rings. $l French)
-	or a single-part expression/work (e.g., $a Tolkien, J.R.R. $t The hobbit. $l French)

We are also not going to be able to tell whether it is for a work or an expression:
-	Since the first example, above, is an aggregating expression/work, the heading is for the aggregating work; but our transform cannot tell this.
-	Since the second and third examples, above, are not aggregating expressions/works, the headings are for expressions; but our transform cannot tell this.

If a headings field is for an **associated WEMI-WEMI relationship (7XXI2 NOT=2)** and has no Expression subfields, then it is not possible to find expression qualifiers in the rest of the MARC record that apply to that heading. So, if we defaulting to mapping the field as an ‘access point for expression’, then we will not have any qualifiers to add to the access point if they are not present in the heading. In which case, the access point for the expression will be the same as the access point for its work.
- 700 1_ $a Tolkien, J.R.R. $t The hobbit.
  - Access point for work: Tolkien, J.R.R. The hobbit
  - Access point for expression: Tolkien, J.R.R. The hobbit. 

And if an expression qualifier is present in a heading, then we must keep it as part of the access point for work in case the described work is an aggregating work. This is problematic for a single expression/work:
- 700 1_ $a Tolkien, J.R.R. $t The hobbit. French
  - Access point for work: Tolkien, J.R.R. The hobbit. French
  - Access point for expression: Tolkien, J.R.R. The hobbit. French

**Question**: Is this at least acceptable for an aggregating work and expression because of the WE lock between them?
    700 1_ $a Lewis, C.S. $t The chronicles of Narnia. $l French
    - Access point for work: Lewis, C.S. The chronicles of Narnia. French 
    - Access point for expression: Lewis, C.S. The chronicles of Narnia. French

If a headings field is for an **aggregated WEMI-WEMI relationship (7XXI2=2)** and has no Expression subfields, then it should be safe (in most cases) to find expression qualifiers in the rest of the MARC record to add as qualifiers to the access point for expression (as long as there is only one value for all embodied expressions/works e.g., one content type or one language). 
- 008 Lang: fre
  700 12 $a Lewis, C.S. $t The chronicles of Narnia
  - Access point for work: Lewis, C.S. The chronicles of Narnia. French
  - Access point for expression: Lewis, C.S. The chronicles of Narnia. French

- 008 Lang: fre
  700 12 $a Lewis, C.S. $t The lion the witch and the wardrobe
  - Access point for work: Lewis, C.S. The lion the witch and the wardrobe. French
  - Access point for expression: Lewis, C.S. The lion the witch and the wardrobe. French 

**Question**: Is any of this workable? Do we have any other options?

**Reply by AdamSchiff** [reply]
I don't think I understand why  Lewis, C.S. $t Chronicles of Narnia. $l French  is an aggregating work access point.  I do not see how the inclusion of $l French connotes anything but an expression.  What am I missing, Deborah?  How can ANY work access point include a language of expression?

Note that LC/PCC NACO policy is to apply the option to omit initial articles in AAPs for works, so all of your examples should not include the article.

**Reply by GordonDunsire** [reply]
@AdamSchiff: None of the examples is for an AAP, just access points. Some authority control systems do not recognize a general access point element; there are only AAPs and VAPs with limited numbers, and I think this reflects pre-computer cataloguing practices where each access point required a separate catalogue card. So in principle leading articles are fine (and crop up in many non-library catalogues). RDA has no objection.

**Reply by tmqdeborah** [reply]
@AdamSchiff 

Lewis, C.S. $t Chronicles of Narnia. $l French is an aggregating work access point because “Chronicles of Narnia” is an aggregating work. 

Because aggregates is not an AACR concept, we will find many 6XX and 7XX headings with expression subfields such as $l and [@pan-zhuo](https://github.com/pan-zhuo) is quite right that the value in the $l becomes a “language of representative expression” for the work (to distinguish the compilation in French from one in English) rather than a “language of expression” for an expression.

**Comment by pan-zhuo** [comment]
I think since aggregating expressions cannot be translated, the "French"
might be taken from "language of representative expression", which is a
work element. Also, "has language version" / "is language version of" are
work-work relationships to connect different language versions of
aggregating works.

But I’ve always wondered what an aggregating work AAP should look like.
C.S. Lewis is not the creator of the French Narnia anymore in the aggregate
model, isn’t he? Why not “[Aggregator name]. Monde de Narnia”? What
about “Lewis,
C.S. Chronicles of Narnia. French. ([Aggregator name])”, if we care about
continuity with earlier practices? Also, can “Lewis, C.S. Chronicles of
Narnia. French” be considered as AAP for the work group of all French
Narnia? But then a work group AAP is also defined as a subproperty of a
work AAP, which means the same work could have 2 different AAPs in a
single vocabulary
scheme???

**Reply by GordonDunsire** [reply]
@pan-zhuo: RDA expects a work group vocabulary to be distinct from a work vocabulary; a work group is not a work or other entity because it is just a 'collect and collocate' device. But in any case RDA does not object to different AAPs in the same scheme.

A consistent general pattern for AAP for a work is {AAP for creator agent of work}. {preferred title of work} ({qualifiers ...}. For an aggregating work, the creator agent is the aggregator, and the creator agent of the aggregated works is one of the potential qualifiers. The language of the aggregated expressions is also a potential qualifier, and is specifically covered in the Toolkit guidance on representative expressions of an aggregating work. For continuity with earlier practice when the aggregated works have the same creator agent (which is part of the plan for aggregation), then

Lewis, C. S. Chronicles of Narnia (French; [AAP for aggregator] is ok.

[AAP for aggregator]. [Preferred title of work] (Lewis, C. S.; French) is also ok.

[Preferred title of work] (Lewis, C. S.; French; AAP for aggregator) is also ok.

They are all just strings :-)

**Reply by tmqdeborah** [reply]
After a discussion with @GordonDunsire , the focus of my original question has moved from undifferentiated Work and Expression access points to returning to the question of what we will use as the default relationships for 6XX, 8XX, and 7XX non-analytical fields.

We had previously agreed that the [default ‘related [WE]’ would be ‘related expression of manifestation’](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-9793794). But if we record a relationship to a related expression, then we really must record a relationship to its work, and, as I explained in the original [Nomen ](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10002990) question, I was concerned that we were not going to be able to differentiate between the access points of the work entities and the expression entities.

Leaving the Nomen question aside, then, let’s move on to [Question: What should we use as the default relationship between a WEMI and an associated WEMI when we have no recorded or mapped RDA URI to use?](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10067307) and revisit our decision.

**Comment by tmqdeborah** [comment]
### Question: What should we use as the default relationship between a described WEMI and an associated WEMI when we have no recorded or mapped RDA URI to use?

As explained in the [original question](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10002990) (re. whether it is OK if an access point for a work has the same Nomen string as an access point for its expression), transformation coding cannot tell whether a heading in a 6XX, 7XX or 8XX is for an aggregating work, a multi-part expression/work, or a single-part expression/work.

If an RDA URI relationship is found, then this distinction should not matter for a relationship between a described WEMI and an ‘**associated**' WEMI. (It does matter for a Primary relationship, but that will be addressed in another discussion.) We should be able to use the relationship to determine both the domain and the range for our triple: reliably for work relationships, not so reliably for expression relationships. 

For example:
700 1  $i Translation of: $a Heller, Carl Benno. $t Jugendstil Máeobel. 
- This is an Expression-Expression relationship, but many such translation examples have no language qualifier for the range Expression. We either must: 
  - find an 041$h (Language code of original) and add it as a language qualifier to the AP for expression 
  - or do not apply RDA URI relationships for Expressions and apply a default relationship instead.

If a valid RDA URI relationship cannot be found (or is undependable), then we need a default WEMI-WEMI relationship. In a previous discussion, we decided that [the default described entity would be Manifestation and the default related entity would be Expression](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-9793794) (‘related expression of manifestation’). 

However, after Gordon and I went over my explanation of the issue, I am now suggesting that the only safe thing that we can do with WEMI relationships is to default to ‘**related work of manifestation**’ instead of ‘related expression of manifestation’. This will mean that:
-  we would only need to record a relationship between the manifestation and the work, without also recording an expression (which meets the requirements of a [minimum description of a work](https://access.rdatoolkit.org/en-US_ala-f4fb013b-b64e-37b1-902c-561d27b1e832/li_rzw_ngd_p2b))
- therefore, we would not need to use the same AP for both the expression and the work.

But, defaulting to ‘related work of manifestation’ also means that:
- the ‘access point for work’ will include any Expression qualifiers that are included in the heading
  - this does not mean that we are saying that the related entity is an Expression, it just means that the access point for the work includes these terms (“[They are all just strings](https://github.com/uwlib-cams/MARC2RDA/discussions/455#discussioncomment-10008698)” –G.D)
  - however, this also means that we cannot use these qualifiers as attributes for our description of the related work.

Do you agree that we should change to using '**related work of manifestation**' instead of 'related expression of manifestation' as our default relationship between a described WEMI and an associated WEMI? **A:** _related work of manifestation_--To be added to "Using" instructions

**Reply by AdamSchiff** [reply]
Maybe this is unorthodox, but why not use the element **_related RDA entity of manifestation_** instead of trying to specify what the particular access point represents when it isn't surely known?  Do we _have_ to pick related work or related expression? Why not go up a level?  It would be completely accurate, if less specific.

**Reply by tmqdeborah** [reply]
Clarification: There is no method for distinguishing between a heading for an aggregating work and a heading for non-aggregating work and we do not **describe** aggregating expressions, therefore we cannot default to Expression--we must default to Work.

Decision (Jul 17, 2024 meeting):
- Use 'related work of manifestation' as our default relationship between a described WEMI and an associated WEMI
- Include (possible) 'expression' qualifiers in the access point for the related work
- Do not map (possible) 'expression' qualifiers as attributes in the description of the related work

**Reply by tmqdeborah** [reply]
Here are some examples. Note that most added entries in older legacy records have no Relationship values ($i or $4. 


700 0  $i Based on (work): $a Dante Alighieri, $d 1265-1321. $t Divina commedia. $l English _– Multi-part work_

700 0  $a Aristotle. $t Rhetoric. $k Selections. $l English. $s (Habinek) $f 2017. $i Container (of work) _– Aggregating work_

700 0  $a Aristophanes. $t Frogs. $l English. $i Adaptation of (work) _– Expression of Single-part work_

700 12 $a Goldberg, Naomi. $t Undoing world. $k Selections _– Aggregating work_

700 0  $a Homer. $t Iliad. $i Adaptation of (work) _– Single-part work_

730 0  $a Census of agriculture (1982). $p Geographic area series. $n Pt. 1-50. _– ???_

700 12 $6 880-07 $a Wu Bruce, Grace, $d 1949- $t Mu qu ju. – _Expression of Single-part work_
700 12 $6 880-08 $a Wu Bruce, Grace, $d 1949- $t Mu qu ju. $l English. – _Expression of Single-part work_

700 12 $a MacManus, Seumas, $d 1869-1960. $t Bewitched fiddle.

700 12 $a Miller, Calvin, $d 1936- $t Symphony in sand.

700 02 $a Averroèes, $d 1126-1198. $t Faòsl al-maqåal fåimåa bayna al-sharåi°ah wa-al-òhikmah min al-ittiòsåal. $p òDamåimah. $l English & Arabic.

245 00 $a Imperial lives and letters of the eleventh century / $c translated by Theodor E. Mommsen and Karl F. Morrison ; with an historical introduction by Karl F. Morrison ; edited by Robert L. Benson.
700 02 $a Wipo, $d active 11th century. $t Gesta Chuonradi II. $l English.
700 02 $a Henry $b IV, $c Holy Roman Emperor, $d 1050-1106. $t Briefe Heinrichs IV. $l English.
_-- Aggregating works_

**Comment by tmqdeborah** [comment]
### Question: How can we tell whether a series added entry applies to an augmented single work vs its aggregating work?

If a manifestation is not an aggregate manifestation, then a series added entry must apply to the work that is described by the main entry of a MARC record. But if a manifestation is an augmentation manifestation, then the series might or might not apply to the work that is described by the main entry

100 1_ |a Austen, Jane, |d 1775-1817.
2451 0 |a Emma : |b an authoritative text : backgrounds, reviews, and criticism / |c Jane Austen ; edited by Stephen M. Parrish.
260 __ |a New York : |b W.W. Norton, |c c1993.
490 0_ |a A Norton critical edition 
830 0_ |a Norton critical edition
  --- Is Austen's work "Emma" in the series or is the aggregating work "Emma" an [issue of ](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-730a0771-a92e-3d32-af1f-10b42fd000da
)the series? (See the definition at the link provided.)

100 1_ |a Penny, Louise.
245 10 |a Bury your dead / |c Louise Penny. 
260 __ |a New York : |b Minotaur Books, |c 2010. 
490 1_ |a Chief Inspector Gamache novel 
800 1_ |a [Penny, Louise. |t Three Pines mystery](https://authorities.loc.gov/cgi-bin/Pwebrecon.cgi?AuthRecID=8020400&v1=2&HC=2&SEQ=20240813125859&PID=KWu8dol1xi0oBwaWK48tPypEteYK). 
   --- Penny's work "Bury your dead" is an issue of this 'author series'

100 1  $aRidgway, Christie.
245 10 $aI still do /$cChristie Ridgway.
440  0 $a[Silhouette special edition ](https://authorities.loc.gov/cgi-bin/Pwebrecon.cgi?AuthRecID=4128830&v1=1&HC=1&SEQ=20240813130447&PID=QloaIVyOTMgFv-h8aR9C-jwZZnU_
);$v1950

245 00 $aHi-lo nonfiction passages for struggling readers :$b80 high-interest/low-readability passages with comprehension questions and mini-lessons for teaching key reading strategies.
440  0 $aTeaching resources

245 10 $aSuicide and justice :$ba Chinese perspective /$cWu Fei.
440  0 $aRoutledge contemporary China series ;$v44
   --- I think this was written for this series, although it could later appear in other series.

100 1  $aSmedman, Lisa.
245 10 $aSacrifice of the widow /$cLisa Smedman.
440  0 $aLady penitent ;$vbk. 1

One possible way to tell if a work is an issue of an author series is when the name portion of a 100 matches the name portion of an 800.

Does anyone know any other ways that we can code to determine which work related to a series added entry.

We will have a similar problem with Reproductions.

What is our fall-back default? 
- When in doubt default to <described Work> **_has related work of work_** <series added entry Work>?


I recommend that we always apply a series added entry to the aggregating work, when one is present. If no aggregating work is present (i.e., the manifestation is NOT an aggregate, not even an augmentation aggregate and the __only__ embodied work is a single expression work) then relate a series added entry to the single expression work. (DF)

**Reply by lake44me** [reply]
What's a series?  How can it be a Work property?  It seems to me a series is something created by a publisher and applied to its published manifestations.  An author series leaves out the middleman, but is still an aspect of how the work was manifested.  Multiple manifestations of a Work (or even an Expression) could have different series.  The series itself could be considered a Work (a successive aggregating one),  but I see having been published in a series as being a manifestation to work relationship.  I'm just not finding properties to express specific series relationships either under Work or Manifestation - what did you have in mind?  http://rdaregistry.info/Elements/m/P30106 Manifestation Series Statement does not permit of recording identifiers or IRIs, only strings for title/title info, numbering, and statement of responsibility.   The only "series" relationships are between a series and a subseries.  I feel like I've fallen through a hole somehow.

**Reply by lake44me** [reply]
@tmqdeborah So addressing your question, the series would need to be applied to the augmentation aggregate manifestation as a whole, wouldn't it?  In whatever relationship we come up with (or just as a series statement, although that seems really lacking in "linked data").  

Feel free to tell me I'm way off here.

**Reply by lake44me** [reply]
Related work of manifestation http://rdaregistry.info/Elements/m/P30265 ?

**Reply by tmqdeborah** [reply]
The relationship between a Work and a series added entry is '[issue of]( https://www.rdaregistry.info/Elements/w/#P10102)'.

After further analysis I recommend that this relationship and its inverse be used only to relate a successive aggregating work (including a serial and a monographic series) to:
- a single expression work 
- an aggregating work

They should not be used as a relationship between any _aggregated_ work (collected, with parallel expressions, or augmented) and a successive aggregating work.
